pkgs/development/libraries/tinyxml-2/default.nix

```diff
@@ -0,0 +1,15 @@
+{ stdenv, fetchurl, cmake }:
+let version = "3.0.0";
+in stdenv.mkDerivation rec {
+  name = "tinyxml-2-${version}";
+  src = fetchurl {
+    url = "https://github.com/leethomason/tinyxml2/archive/${version}.tar.gz";
+    sha256 = "0ispg7ngkry8vhzzawbq42y8gkj53xjipkycw0rkhh487ras32hj";
+  };
+
+  configurePhase = ''
+    cmake -DCMAKE_INSTALL_PREFIX=$out .
```

Comment:
I think it used to be necessary because I didn't know about nativeBuildInputs, so the cmake envHooks never ran. I will confirm and resubmit.